src/Microsoft.WSMan.Management/WsManHelper.cs

```diff
@@ -1098,10 +1098,10 @@ internal static void LoadResourceData()
                         string Line = _sr.ReadLine();
                         if (Line.Contains("="))
                         {
-                            string[] arr = Line.Split(new char[] { '=' }, 2);
+                            string[] arr = Line.Split('=', 2);
```

Comment:
Might be worth labelling the `2` parameter

Reply:
Fixed.